.github/workflows/check-outdated-trusted-root.yml

```diff
@@ -0,0 +1,20 @@
+name: "Check for outdated trusted root certificates"
+
+on:
+  schedule:
+    - cron: "0 0 * * 2"
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Pull new trusted root
+        run: gh attestation trusted-root > resources/new-trusted-root.jsonl
+      - name: Create an issue if different
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          diff resources/trusted-root.jsonl resources/new-trusted-root.jsonl \
+            && echo "Trusted root cert has not changed, no action required." \
+            || gh issue create -R php/pie --title "Trusted root needs updating" --body "The trusted root certificate file needs updating. Use the \`gh attestation trusted-root > resources/trusted-root.jsonl\` command to update it." --assignee "asgrim"
```

Comment:
Need to add a check if there is already an open issue with same name to avoid re-opening